frontend/next.config.js

```diff
@@ -15,7 +15,9 @@ if (process.env.NEAR_NETWORKS) {
       name: "localhostnet",
       explorerLink: "http://localhost:3000",
       aliases: ["localhost:3000", "localhost", "127.0.0.1", "127.0.0.1:3000"],
-      nearWalletProfilePrefix: "https://wallet.near.org/profile",
+      nearWalletProfilePrefix:
+        process.env.NEAR_WALLET_PROFILE_URL ||
+        "https://wallet.near.org/profile",
```

Comment:
I suggest you set `NEAR_NETWORKS` environment variable (it should just have a list of objects of `name`, `explorerLink`, `aliases`, and `nearWalletProfilePrefix`; I assume you only need a single object in that list), and don't bother with the introduction of a new environment variable.